chef/cookbooks/horizon/recipes/server.rb

```diff
@@ -165,10 +167,13 @@
 unless monasca_ui_pkgname.nil?
   monasca_servers = search(:node, "roles:monasca-server") || []
   unless monasca_servers.empty?
+    include_recipe "#{@cookbook_name}::monasca_ui"
     package monasca_ui_pkgname do
       action :install
       notifies :reload, resources(service: "apache2")
     end
+    grafana_available = true
```

Comment:
Where is `grafana_url` coming from? Maybe we can use it, but `monasca_servers.empty?` is a very reliable test. In doubt I'd go for using that as a drop-in replacement for `grafana_available` rather than the contents of `grafana_url` (I agree that `grafana_available` itself is rather redundant).